cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/FieldOperationInfo.java

```diff
@@ -16,29 +16,28 @@
 
 package co.cask.cdap.proto.metadata.lineage;
 
+import co.cask.cdap.api.annotation.Beta;
+
 /**
- * Represents field lineage operation.
+ * Represents field lineage operation. Each individual operation
```

Comment:
Represents an operation in the lineage for a field. An operation is given by its name and description, along with the inputs it operates on and the outputs it emits.

Reply:
done.